src/init.lua

```diff
@@ -2504,13 +2504,17 @@ end
 xplr.fn.builtin.fmt_general_table_row_cols_1 = function(m)
   local r = m.tree .. m.prefix
 
+  local function path_escape(path)
```

Comment:
I'm wondering if this can be removed and the backend provide a `m.relative_path_escaped` that uses the rust module I added instead.